testing/tests/DevExpress.ui.widgets.dataGrid/headerFilter.tests.js

```diff
@@ -27,6 +27,18 @@ import messageLocalization from 'localization/message';
 import dateSerialization from 'core/utils/date_serialization';
 import { ListSearchBoxWrapper } from '../../helpers/wrappers/searchBoxWrappers.js';
 
+function getListOrTreeView(columnIndex) {
+    if(columnIndex !== undefined) {
+        this.headerFilterController.showHeaderFilterMenu(columnIndex);
+    }
+
+    const $popupContent = this.headerFilterView.getPopupContainer().$content();
+    const list = $popupContent.find('.dx-list');
+    const treeView = $popupContent.find('.dx-treeview');
+
+    return list.length ? list.dxList('instance') : treeView.dxTreeView('instance');
+}
+
```

Comment:
Minor, maybe it's better to leave as is: it's not obvious that function with name like 'get...' not only getting some value, but performs action (showing filter menu in this case)